Deploy-CAM.ps1

```diff
@@ -2519,6 +2519,57 @@ function Confirm-ModuleVersion()
     return $true
 }
 
+# Create a custom role for CAM with necessary permissions
+# Currently it is the based off of the Contributor Role excpept we remove Storage Access and VM Disk Blob Read Access
+# Use 'Get-AzureRmProviderOperation *' to get a list of Azure Operations and their details
+# See https://docs.microsoft.com/en-us/azure/active-directory/role-based-access-built-in-roles for details on Azure Built in Roles
+function Get-CAMRoleDefinition() {
+    param(
+        [parameter(Mandatory = $true)]
+        [String]$subscriptionId
+    )
+
+    $camCustomRoleDefinition = Get-AzureRmRoleDefinition "Cloud Access Manager"
```

Comment:
What happens if the account doesn't have the correct access rights for the 'roledefinition' API's? Uncaught exception? Other badness?

Reply:
Yeah, it throws an permissions error.

Reply:
Okay, as long as the error is pretty clear and things exit, sounds good.